app/tests/components/Form/ReadOnlySubmissionWidget.test.tsx

```diff
@@ -0,0 +1,52 @@
+import FormTestRenderer from '../../utils/formTestRenderer';
+import { render, screen } from '@testing-library/react';
+import type { JSONSchema7 } from 'json-schema';
+
+const schema = {
+  title: 'Read only submission widget test',
+  type: 'object',
+  required: ['submissionDate'],
+  properties: {
+    submissionDate: {
+      title: 'On this date (YYYY-MM-DD)',
+      type: 'string',
+    },
+  },
+};
+
+const uiSchema = {
+  submissionDate: {
+    'ui:widget': 'ReadOnlySubmissionWidget',
+  },
+};
+
+const renderStaticLayout = (
+  schema: JSONSchema7,
+  uiSchema: JSONSchema7,
+  formData
+) => {
+  return render(
+    <FormTestRenderer
+      formData={formData}
+      onSubmit={() => console.log('test')}
+      schema={schema as JSONSchema7}
+      uiSchema={uiSchema}
+    />
+  );
+};
+
+describe('The ReadOnlySubmissionWidget', () => {
+  it('should render the title', () => {
+    renderStaticLayout(schema as JSONSchema7, uiSchema as JSONSchema7, {});
```

Comment:
Instead of a cast here, typing the `schema` variable above would give you better typing. The cast will effectively silence the type checking.